intelligent-video-recording/gst_display.py

```diff
@@ -47,24 +57,32 @@ def __init__(self, config_file_name):
                 mobilenetv2 name=net labels=\"%s\" model-location=%s backend=coral \
                 backend::input-layer=%s backend::output-layer=%s \
                 net.src_bypass ! inferenceoverlay ! videoconvert ! \
-                autovideosink name=videosink sync=false " % \
+                clockoverlay valignment=bottom halignment=right ! \
+                interpipesink name=inference_src sync=false" % \
                 (video_dev,labels,model,input_layer,output_layer)
 
-        # Create GStreamer pipeline
-        self.pipeline = Gst.parse_launch(pipe)
+        pipe2 = "interpipesrc name=display_sink listen-to=inference_src ! \
```

Comment:
Don't call it pipe2, you can call it display or display_pipe, a more representative name